storage/sql.go

```diff
@@ -352,10 +353,13 @@ func verifyModel(ctx context.Context, db *pg.DB, schemaName string, m *orm.Table
 		}
 
 		// Some common aliases
-		if datatype == "timestamp with time zone" {
+		switch datatype {
+		case "timestamp with time zone":
+			fallthrough
+		case "timestamp without time zone":
 			datatype = "timestamptz"
-		} else if datatype == "timestamp without time zone" {
-			datatype = "timestamp"
+		case "ARRAY":
+			datatype = "bigint[]"
```

Comment:
why is this needed?

Reply:
Because of this: https://github.com/filecoin-project/lily/blob/master/storage/sql_test.go#L49 -- it's a convenience function being used in such a test.

Moreover, the mapping of data types between Postgres and Golang is not one-to-one [that's why the original code has this condition as well](https://github.com/filecoin-project/lily/blob/master/storage/sql.go#L355-L359) because it tries to equate a Postgres datatype with a `fld.SQLType` [here](https://github.com/filecoin-project/lily/blob/master/storage/sql.go#L361-L363)